mesonbuild/backend/backends.py

```diff
@@ -594,6 +614,20 @@ def get_executable_serialisation(
             exe_wrapper = None
 
         workdir = workdir or self.environment.get_build_dir()
+
+        needs_rsp_file = can_use_rsp_file and sum(len(i) for i in cmd_args) >= get_rsp_threshold()
```

Comment:
Should the command length take into account spaces between arguments?

Reply:
Yes, they count towards the 32k limit as well.